src/services/PlacesService.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
```

Comment:
This service also already exists and name of service in lowercase please merge your branch with `main`